src/routes/categories/containers/__tests__/CategoriesPage.spec.js

```diff
@@ -81,7 +81,7 @@ describe('CategoriesPage', () => {
 
   const language = 'en'
 
-  test('should render a Page if page has no children', () => {
+  test('should match snapshot and render a Page if page has no children', () => {
     const mockSetLanguageChangeUrls = jest.fn()
     const mockReplaceUrl = jest.fn()
```

Comment:
You don't need to mock functions if you don't use the mock. Just pass a () => {}

Reply:
done